backend/pkg/db/db.go

```diff
@@ -30,11 +30,6 @@ type Repository interface {
 	UpdateBackend(ctx context.Context, backend *model.Backend) (*model.Backend, error)
 	GetBackend(ctx context.Context, id string) (*model.Backend, error)
 	ListBackend(ctx context.Context, limit, offset int, query interface{}) ([]*model.Backend, error)
-	CreateTier(ctx context.Context, tier *model.Tier) (*model.Tier, error)
```

Comment:
Delete the implementations of these from mongo.go file, that is the reason your build is failing

Reply:
Corrected